js/data.js

```diff
@@ -953,7 +1084,7 @@ var data = {
         kennung: [SSKKO, SSKMA],
         lehrform: ["Seminar"],
         modul: ["Softskills"],
-        semester: [ss13,ws13_14, ss14, ws14_15, ss15, ws15_16, ss16],
+        semester: [ss13,ws13_14, ss14, ws14_15, ss15, ws15_16, ss16, ws16_17],
         cp: 3,
         dozent: ["Dr. Michaela Schumacher", "Eva Bilstein"],
```

Comment:
I'll correct this. I am not sure if the previous lectures were held by Dr. Schumacher. I did not wrote that line.